modules/popup/react/lib/usePopupStack.ts

```diff
@@ -40,23 +41,22 @@ import {PopupStack} from '@workday/canvas-kit-popup-stack';
  * useCloseOnEscape(stackRef, closePopup)
  */
 export const usePopupStack = <E extends HTMLElement>(
-  forwardRef?: React.RefObject<E>,
+  ref?: React.Ref<E>,
   target?: HTMLElement | React.RefObject<HTMLElement>
-): React.RefObject<HTMLDivElement> => {
-  const internalRef = React.useRef<HTMLDivElement>(null);
-  const ref = (forwardRef || internalRef) as React.RefObject<HTMLDivElement>;
+): React.RefObject<HTMLElement> => {
+  const {elementRef, localRef} = useLocalRef(ref);
```

Comment:
`useLocalRef` is a convenience wrapper around `useForkRef` that extracts types from the provided ref and returns accurate types that are more clear on the intent.